### DISCUSSION TITLE
usb audio device on imxrt1011 devboard

### DISCUSSION THREAD

**Discussion by cdg66** [hathach/tinyusb#579] [Q&A]
Hello,

For a project for my final exam i want to take an USB audio stream from a computer and output it to an i2s interface in order to be played. I'll be using an imxrt1011 devboard and MCUExpresso ide for that project. I'm new to the tinyUSB stack and i need some help to set it up and make it work. I've followed your getting started tutorial but i dont really know what im doing and im lost. For example do i call tud_int_handler() in the IRQ of the USB PHY or the one of the USB OTG. Thanks for your help.

**Comment by hathach** [comment]
have you tried to compile and run the `examples/device/audio_test` and `examples/device/uac2_headset`. Looking at running example could give you some hint.
